pulp_ansible/app/tasks/utils.py

```diff
@@ -43,7 +43,15 @@ def parse_metadata(download_result):
         return json.load(fd)
 
 
-RequirementsFileEntry = namedtuple("RequirementsFileEntry", ["name", "version", "source"])
+class RequirementsFileEntry(namedtuple("RequirementsFileEntry", ["name", "version", "source"])):
+    """Object to store collections from requirements file."""
+
+    def included(self, other):
+        """Simple inclusivity test, check if this RFE is part of another."""
+        if self.name == other.name:
+            if other.version == "*":
+                return True
+        return False
```

Comment:
This should help reduce api requests for collections with shared dependencies when using paginated metadata endpoints.